### PR TITLE
Feature/motor control

### PR DESCRIPTION
**Pull request by parasharmohapatra**
added 2 new files, motor.py and testing_class.py

### CODE REVIEW

**Review by raphaelcervantes — COMMENTED** [review]
@parashar2399 Looks good. I will merge after I test out these functions on Monday.

If you wanted to make this code follow better practices, I would recommend adding docstrings to classes and functions. For example, see this dripline code here https://github.com/driplineorg/dripline-python/blob/33532c3b0eff14389e4e587cdf99c00d92b208ac/dripline/implementations/ethernet_scpi_service.py#L18

Also, you can read about docstrings more general here: https://www.python.org/dev/peps/pep-0257/

But I will just merge to master regardless of whether or not you add docstrings.

**Comment by raphaelcervantes** [comment]
@parashar2399 I want to see these motor classes in action. Can you rewrite modemap.py (or motor_control.py) using these functions by 2 PM today? I can then run your script while observering the resonator. Otherwise, I can test out your script tomorrow.

File: data_taking_scripts/dripline_logger.py
Comment (raphaelcervantes) [inline data_taking_scripts/dripline_logger.py:9]: @parashar2399 this function is not named properly. This does not take the entire modemap measurement. It just takes a logs a single s11 and a single s21.

File: data_taking_scripts/dripline_logger.py
Comment (raphaelcervantes) [inline data_taking_scripts/dripline_logger.py:3]: @parashar2399 The name of this class is way too generic. This class is pretty specific to logging S21 and S11 for a modemap measurement. 
https://github.com/driplineorg/dripline-python/blob/master/dripline/implementations/postgres_sensor_logger.py 
and
https://github.com/driplineorg/dripline-python/blob/master/dripline/implementations/postgres_interface.py
Are for more generic dripline logging.

File: data_taking_scripts/motor.py
Comment (raphaelcervantes) [inline data_taking_scripts/motor.py:154]: @parashar2399 After fixing some class function calls, I finally got to this error. I'm not sure what is happening, so I leave it to you to fix it. My suggestion is that once you fix it, you try to take a modemap where you just move the resonator motor. 

```
root@clerk-dripline-python-deployment-7c8cff556c-trscb:/usr/local/src/dripline-python-plugin/data_taking_scripts# python testing_class.py                                                                                                                                     
fail!
fail!
Enter initial resonator length (in cm): 16
Enter the distance to move in cm (Empty cavity modemap is usually 3): 3
Enter the number of measurements needed: 10
done waiting
Starting modemap measurement
Setting na_measurement_status to start_measurement
Logging list of endpoints
Setting na_measurement_status to stop_measurement
Traceback (most recent call last):
  File "testing_class.py", line 42, in <module>
    num_plates)
  File "/usr/local/src/dripline-python-plugin/data_taking_scripts/motor.py", line 108, in move_by_increment
    curved_mirror_steps = self.curved_mirror_distance_to_steps(increment_distance)
  File "/usr/local/src/dripline-python-plugin/data_taking_scripts/motor.py", line 147, in curved_mirror_distance_to_steps
    num_pitch_lengths = distance/pitch #these many complete rotations
TypeError: unsupported operand type(s) for /: 'OrpheusMotors' and 'float'

```

**Review by raphaelcervantes — CHANGES_REQUESTED** [review]
@parashar2399 
I pushed some changes as I was trying to see what I could fix on my own. But there is a problem with curved_mirror_distance_to_steps that I don't want to deal with right now. Can you debug it and test your changes on the testing motor? S21 and S11 won't look like they are changing as a function of frequency because we're not moving the curved mirror. But that's fine!

File: data_taking_scripts/dripline_logger.py
Comment (parasharmohapatra) [reply data_taking_scripts/dripline_logger.py:3]: I was thinking that we could use it for all our data logging functions. But I can change it if you'd like. 
I'll go ahead and change the name of log_modemap for now

**Comment by raphaelcervantes** [comment]
It seemed to run ok. I'm going to merge now. More work will probably be needed later, and that's fine.

**Pull request merged**
